trade-rules.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
should trade-rules be top level or could it go in a folder?  should it go in `local` like the api-keys, or is it better to check it into the repo somewhere?

Reply:
I think it's handy to have it checked in as an example/defaults for other users, plus having it always defined avoids having to do another error handling check at startup.